after/ftplugin/php.vim

```diff
@@ -20,6 +20,14 @@ if !exists('g:hack#hh_client')
   let g:hack#hh_client = 'hh_client'
 endif
 
+if !exists('g:hack#hh_format') && executable('hh_format')
+  let g:hack#hh_format = 'hh_format'
+endif
+
+if !exists('g:hack#root')
+  let g:hack#root = '~/www'
+endif
```

Comment:
Yeah please don't hardcode `~/www` anywhere in this plugin, even as a fallback default. It's emphatically not a reasonable default anywhere but FB.